src/components/main/MainPage.jsx

```diff
@@ -24,26 +26,56 @@ class MainPage extends Component {
       });
   }
 
+  countingItems = (checklist) => {
+    if (checklist.length === 1) {
+      return `${checklist.length} item`;
+    }
+    return `${checklist.length} items`;
+  }
+
   render() {
     const { showAllCheckList, loading } = this.state;
+    console.log(showAllCheckList);

```

Comment:
remove console.log

Reply:
done, can u check pls how i got author because we should change the concept of giving author value